pkg/solidity-utils/contracts/helpers/FactoryWidePauseWindow.sol

```diff
@@ -14,6 +16,8 @@ pragma solidity ^0.8.24;
  * When the buffer period expires, it will unpause automatically, and remain permissionless forever after.
  */
 contract FactoryWidePauseWindow {
+    using SafeCast for *;
```

Comment:
Hmm where are we using this?

Reply:
Removed... it *was* used to case the timestamp in the `getNewPoolPauseWindowEndTime` function, but then I realized that would revert when it overflowed (in 2106, but still...), but the inequality would still work, so as it is now it will return 0 after 2106 instead of reverting, as it should. No longer needed the cast, but forgot to remove it.